assets/tests/components/map/controls/layersControl.test.tsx

```diff
@@ -63,4 +74,97 @@ describe("LayersControl", () => {
 
     expect(screen.queryByLabelText("Map (default)")).toBeNull()
   })
+
+  test("pull-back layer control not shown when user is not in test group", async () => {
+    ;(getTestGroups as jest.Mock).mockReturnValue([])
+
+    render(<LayersControl tileType="base" setTileType={jest.fn()} />, {
+      wrapper: mapWrapper,
+    })
+    await userEvent.click(layersControlButton.get())
+
+    expect(pullbacksSwitch.query()).not.toBeInTheDocument()
+  })
+
+  test("pull-back layer control is shown when user is in test group", async () => {
+    ;(getTestGroups as jest.Mock).mockReturnValue(["pull-back-map-layer"])
+
+    render(
+      <LayersControl
+        tileType="base"
+        setTileType={jest.fn()}
+        pullbackLayerEnabled={false}
+        togglePullbackLayerEnabled={jest.fn()}
+      />,
+      {
+        wrapper: mapWrapper,
+      }
+    )
+    await userEvent.click(layersControlButton.get())
+
+    expect(pullbacksSwitch.get()).toBeInTheDocument()
+  })
+
+  test("clicking pull-back layer control toggles pull-back layer", async () => {
+    ;(getTestGroups as jest.Mock).mockReturnValue(["pull-back-map-layer"])
+
+    const mockToggle = jest.fn()
+
+    render(
+      <LayersControl
+        tileType="base"
+        setTileType={jest.fn()}
+        pullbackLayerEnabled={false}
+        togglePullbackLayerEnabled={mockToggle}
+      />,
+      {
+        wrapper: mapWrapper,
+      }
+    )
+    await userEvent.click(layersControlButton.get())
+    await userEvent.click(pullbacksSwitch.get())
+
+    expect(mockToggle).toHaveBeenCalledTimes(1)
+  })
+
+  test("enter key toggles pull-back layer", async () => {
+    ;(getTestGroups as jest.Mock).mockReturnValue(["pull-back-map-layer"])
+
+    const mockToggle = jest.fn()
+
+    render(
+      <LayersControl
+        tileType="base"
+        setTileType={jest.fn()}
+        pullbackLayerEnabled={false}
+        togglePullbackLayerEnabled={mockToggle}
+      />,
+      {
+        wrapper: mapWrapper,
+      }
+    )
+    await userEvent.click(layersControlButton.get())
+    pullbacksSwitch.get().focus()
+    await userEvent.keyboard("{Enter}")
+
+    expect(mockToggle).toHaveBeenCalledTimes(1)
+  })
+
+  test("when pull-back layer is enabled, shows pill", async () => {
```

Comment:
suggestion (non-blocking): Maybe another test that the pill is hidden when pull-back layer is not enabled?